tests/test_logger.py

```diff
@@ -49,3 +52,18 @@ class A(object):
         args.verbose = True
         CmdBase._set_loglevel(args)
         self.assertEqual(Logger.logger().getEffectiveLevel(), logging.DEBUG)
+
+    @patch('sys.stderr.write')
+    @patch('sys.stdout.write')
+    def test_stderr(self, mock_stdout, mock_stderr):
+        main(['config', 'non_existing_section.field'])
```

Comment:
I guess it wouldn't hurt to check main() return code just to be extra sure. Could you please add an assert on return code being 0?